libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -224,6 +224,9 @@ class AP_Motors {
                     PWM_TYPE_PWM_RANGE  = 8 };
     pwm_type            get_pwm_type(void) const { return (pwm_type)_pwm_type.get(); }
 
+    // return true if a PWM type is a DShot varient
```

Comment:
I think it is "variant" with an "a" although obviously not a blocker.  otherwise looks good.